ui/app/components/auth-form.js

```diff
@@ -255,9 +272,7 @@ export default Component.extend(DEFAULTS, {
       yield timeout(1000);
```

Comment:
It's a little hard for me to remember/understand the flow here but this is another yield of one second. Is there a situation where they could wait 1 second and then have to wait another second? Is that an issue at all for the okta flow is that okay?

Reply:
We yield for 1s there because we need to keep polling /auth/okta/verify/:nonce API every 1s until a response is given with the correct number for the Okta Number Challenge. In that loop we're waiting for the user to click "Yes its me" on their phone so we can receive the correct answer for the Okta Number Challenge. I think it is more that we are waiting for the user to respond to the push notification, but yes there sometimes is a 1s delay after they respond to the push notification before we receive the correct answer and display it.